src/test/java/serenitylabs/tutorials/trains/WhenPlanningATrip.java

```diff
@@ -1,14 +1,47 @@
 package serenitylabs.tutorials.trains;
 
 import net.serenitybdd.junit.runners.SerenityRunner;
+import net.serenitybdd.screenplay.Actor;
+import net.serenitybdd.screenplay.abilities.BrowseTheWeb;
+import net.thucydides.core.annotations.Managed;
+import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
+import org.openqa.selenium.WebDriver;
+import serenitylabs.tutorials.trains.questions.TheOutBoundJourneySummary;
+import serenitylabs.tutorials.trains.tasks.tasks.ChosenTo;
+import serenitylabs.tutorials.trains.tasks.tasks.ViewAvailableTickets;
+
+import static net.serenitybdd.screenplay.GivenWhenThen.seeThat;
+import static org.hamcrest.CoreMatchers.is;
 
 @RunWith(SerenityRunner.class)
 public class WhenPlanningATrip {
+    @Managed(driver = "chrome")
+    static WebDriver webDriver ;
+
+    Actor tracy;
+
+
+    @Before
+    public void setTheStage() {
+        System.setProperty("webdriver.chrome.driver", "D:\\Users\\sapurani\\Desktop\\Chrome Driver\\chromedriver.exe");
```

Comment:
Hi Saket,

I think we should not mention SetProperty or anything else as we are doing @Manged  annotation over WebDriver. So it will take care/manage lifecycle of webdriver from start to end.

Pls confirm once from John as well.